mysql-test/suite/sys_vars/r/default_table_encryption_basic.result

```diff
@@ -139,13 +141,15 @@ SELECT @@global.default_table_encryption;
 @@global.default_table_encryption
 ON
 SET PERSIST default_table_encryption=ONLINE_TO_KEYRING;
+ERROR 42000: Variable 'default_table_encryption' can't be set to the value of 'ONLINE_TO_KEYRING'
```

Comment:
why it doesn't print that keyring plugin is not loaded? Specially this message
```
          thd, Sql_condition::SL_WARNING, ER_WRONG_ARGUMENTS,
                        " (innodb_undo_log_encrypt) and try again.");
          "The default_table_encryption option cannot be changed, "
    return true;
          "keyring plugin is not available");
```

Reply:
This is done similar to other checks in that method now. The message goes to warnings and can be seen with 'show warnings'. I'll take a look how we could change error message in this case.

Reply:
Please check. "cannot be set" is not user friendly. We suggest an action that user can execute. If you cannot reuse this existing message, add "Please check keyring plugin is loaded" (or something on that lines). Also, add "SHOW WARNINGS".

Reply:
> "cannot be set"
This one comes from standard error messages

> ER_WRONG_VALUE_FOR_VAR 42000
>        eng "Variable '%-.64s' can't be set to the value of '%-.200s'"

Reply:
Right :disappointed: ! This is for uses case when user sets wrong values.. (like out of the boundary, string). Doesn't need an explanation. 

Maybe use different ER_ code for ONLINE_KEYRING mode failures?

Reply:
Updated it a bit, now it prints "keyring plugin is not available" error.